src/quo2/components/list_items/received_contact_request.cljs

```diff
@@ -0,0 +1,79 @@
+                      :padding-bottom 12}}
+     [user-avatar/user-avatar {:full-name         display-name
+                               :status-indicator? true
+                               :online?           true
+                               :size              :small
+                               :profile-picture   image
+                               :ring?             false}]
+     [rn/view {:style {:margin-horizontal 8}}
+      [rn/view {:style {:flex-direction :row}}
+       [rn/text {:style (merge typography/font-semi-bold typography/paragraph-1
+                               {:color        (colors/theme-colors "#000000" "#ffffff")
+                                :margin-right 8})} display-name]
+       (when no-ens-name [rn/text {:style (merge typography/font-regular typography/label
+                                                 {:color      (colors/theme-colors colors/neutral-50 colors/neutral-40)
+                                                  :margin-top 4})}
+                          (str (utils.utils/get-shortened-address chat-id) " · ")])
+       [rn/text {:style (merge typography/font-regular typography/label
+                               {:color      (colors/theme-colors colors/neutral-50 colors/neutral-40)
+                                :margin-top 4})}
+        (time/to-short-str timestamp)]]
+      [rn/view {:style {:border-radius      12
+                        :margin-top         10
+                        :padding-horizontal 12
+                        :padding-vertical   8
+                        :border-width       1
+                        :border-color       (colors/theme-colors colors/neutral-20 colors/neutral-70)}}
+       [rn/text {:style (merge typography/font-regular
+                               typography/paragraph-1
+                               {:color (colors/theme-colors "000000" "#ffffff")})}
+        (:text (:content message))]]
+      [rn/view {:style {:margin-top     12
+                        :flex-direction :row}}
+       [rn/touchable-opacity {:accessibility-label :decline-cr
+                              :on-press            #(>evt [:contact-requests.ui/decline-request (:message-id message)])
+                              :active-opacity      1
+                              :style               {:background-color   (colors/theme-colors colors/danger-50 colors/danger-60)
+                                                    :justify-content    :center
+                                                    :align-items        :center
+                                                    :align-self         :flex-start
+                                                    :border-radius      8
+                                                    :padding-vertical   4
+                                                    :padding-horizontal 8}}
+        [rn/text {:style (merge typography/font-medium typography/paragraph-2 {:color "#ffffff"})} (i18n/label :t/decline)]]
```

Comment:
Is this white color going to work well for light and dark modes? I'm just asking this because other parts of this function are relying on `colors/theme-colors`.

Reply:
Yes, it is white in light and dark mode, as the button background is dark.

Reply:
yep, there is a white in the colors file, so you should use `colors/white` here